src/webapp/models.py

```diff
@@ -565,6 +565,17 @@ def get_translation_dict():
     except Exception as e:
         raise Exception("Could not read Labels table: " + str(e))
 
+def get_norwegian_to_english_dict():
+    """
+        Reads all labels from database and create dictionary
+        that translates from norwegian to english.
+    """
+    try:
+        labels = Labels.query.all()
```

Comment:
Not sure if it's a good idea quering for all labels: Made another query that works pretty good and only returns the relevant label:

```
def to_english(norwegian_label):
    """
        Reads the labels table and return the english primary key of the norwegian word
    """
    try:
        query = Labels.query.filter(Labels.norwegian == norwegian_label)[0]
        return str(query.english)

    except AttributeError as e:
        raise AttributeError(
            "Could not find translation in Labels table: " + str(e)
        )
```

Reply:
Should be fine as long as we have a one-to-one mapping in the db which is the case for now and probably looking forward as well

Reply:
Made the changes now. And it worked perfectly. Good suggestion!